lib/get-exports.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const getEsmExports = require('./get-esm-exports.js')
+const { parse: getCjsExports }= require('cjs-module-lexer')
```

Comment:
I'm surprised `}=` slipped through the linter

Reply:
Oh, looks like `lib/` is a new directory and needs to be added to the linter rules?

Reply:
Whoops, wrong repo.